gdx-jnigen/src/main/java/com/badlogic/gdx/jnigen/BuildTarget.java

```diff
@@ -161,49 +174,106 @@ public static BuildTarget newDefaultTarget (Os type, boolean is64Bit, boolean is
-					"-c -Wall -O2 -fmessage-length=0",
-					"-Wl,--kill-at -shared -static -static-libgcc -static-libstdc++");
-			target.architecture = Architecture.ARM;
-			return target;
-		}
+	/** Creates a new default BuildTarget for the given OS, using common default values. */
+	public static BuildTarget newDefaultTarget (Os type, Architecture.Bitness bitness, Architecture architecture, CompilerABIType abiType) {
+
+        if (abiType == CompilerABIType.MSVC) {
+            if (type == Os.Windows && architecture == Architecture.x86 && bitness == Architecture.Bitness._32) {
+                // Windows x86 32-Bit
+                BuildTarget target = new BuildTarget(Os.Windows, Architecture.Bitness._32, new String[]{"**/*.c"}, new String[0], new String[]{"**/*.cpp"},
+                        new String[0], new String[0], "", "/O2 /W4 /fp:fast /arch:IA32",
+                        "/O2 /W4 /fp:fast /arch:IA32",
+                        "/DLL", "/MT");
+				target.cCompiler = "cl.exe";
+				target.cppCompiler = "cl.exe";
+				target.compilerABIType = abiType;
+                return target;
+            }
+
+            if (type == Os.Windows && architecture == Architecture.x86 && bitness == Architecture.Bitness._64) {
+                // Windows x86 64-Bit
+				BuildTarget target = new BuildTarget(Os.Windows, Architecture.Bitness._64, new String[]{"**/*.c"}, new String[0], new String[]{"**/*.cpp"},
+                        new String[0], new String[0], "", "/O2 /W4 /fp:fast /arch:AVX2",
+                        "/O2 /W4 /fp:fast /arch:AVX2",
+                        "", "/MT");
+				target.compilerABIType = abiType;
+				target.cCompiler = "cl.exe";
+				target.cppCompiler = "cl.exe";
+				return target;
+            }
+
+            if (type == Os.Windows && architecture == Architecture.ARM && bitness == Architecture.Bitness._32) {
+                // Windows ARM 32-Bit
+                BuildTarget target = new BuildTarget(Os.Windows, Architecture.Bitness._32, new String[]{"**/*.c"}, new String[0], new String[]{"**/*.cpp"},
+                        new String[0], new String[0], "", "/O2 /W4 /fp:fast /arch:IA32",
+                        "/O2 /W4 /fp:fast /arch:IA32",
+                        "", "/MT");
+				target.cCompiler = "cl.exe";
+				target.cppCompiler = "cl.exe";
+                target.architecture = Architecture.ARM;
+				target.compilerABIType = abiType;
+                return target;
+            }
+
+            if (type == Os.Windows && architecture == Architecture.ARM && bitness == Architecture.Bitness._64) {
+                // Windows ARM 64-Bit
+                BuildTarget target = new BuildTarget(Os.Windows, Architecture.Bitness._64, new String[]{"**/*.c"}, new String[0], new String[]{"**/*.cpp"},
+                        new String[0], new String[0], "", "/O2 /W4 /fp:fast /arch:ARM64",
+                        "/O2 /W4 /fp:fast /arch:ARM64",

```

Comment:
Shouldn't the arch attribute be something from this list: `/arch:<armv8.0|armv8.1|armv8.2|armv8.3|armv8.4|armv8.5|armv8.6|armv8.7|armv8.8|armv8.9|armv9.0|armv9.1|armv9.2|armv9.3|armv9.4>`?(https://learn.microsoft.com/en-us/cpp/build/reference/arch-arm64?view=msvc-170)